lib/features/family/features/reflect/presentation/pages/summary_screen.dart

```diff
@@ -27,12 +28,25 @@ class _SummaryScreenState extends State<SummaryScreen> {
   final _cubit = getIt<SummaryCubit>();
   bool pressDown = false;
 
+  @override
+  void initState() {
+    super.initState();
+    WakelockPlus.enable();
+  }
```

Comment:
Isn't this already on dev? Some github  refresh bug?

Reply:
Had to pull dev into this branch indeed